config/standalone/kv-cache-controller/patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
```

Comment:
for patches.yaml, it will append aibrix as well. Right?

Reply:
yeah, here it defines the namespace, the configuration here will be overwritten. 

![image](https://github.com/user-attachments/assets/24d689f9-ad76-4e1e-99dd-5b39156f9aaa)